corehq/apps/app_manager/views/apps.py

```diff
@@ -340,6 +340,11 @@ def get_apps_base_context(request, domain, app):
             or getattr(app, 'commtrack_enabled', False)
         )
 
+        show_biometric = (
+            toggles.BIOMETRIC_INTEGRATION.enabled(domain)
```

Comment:
Doesn't `app.is_biometric_enabled` already check the flag?

Reply:
I think we want to be able to override that `is_enabled` flag with a permission once we decide what plan this should be on, hence the toggle. right now we'll control via the toggle